test/Interop/Cxx/class/Inputs/sendable.h

```diff
@@ -0,0 +1,27 @@
+struct HasPrivatePointerField {
```

Comment:
Could we add a test case that has inheritance?

Reply:
Yeap, added a test